snap/snapcraft.yaml

```diff
@@ -31,8 +28,17 @@ parts:
       sed -e '1c#!/usr/bin/env python3' -i $CRAFT_PART_INSTALL/snap/blanket/current/usr/bin/blanket
     organize:
       snap/blanket/current: .
+
+  gpu-2404:
+    after: [blanket]
+    source: https://github.com/canonical/gpu-snap.git
+    plugin: dump
+    override-prime: |
+      craftctl default
+      ${CRAFT_PART_SRC}/bin/gpu-2404-cleanup mesa-2404
     prime:
-      - -snap/blanket
+    - bin/gpu-2404-wrapper
```

Comment:
```suggestion
    - -*
```

Reply:
Hmm... with this change, I'm getting stuck in the "Installing build-snaps" phase... it is currently at 2600 seconds and still running.

Reply:
Ok, it took a while, but with the latest version it builds (still with a few linter warnings), and runs properly.

Let me know if you want any other changes.

Reply:
Nope, let's merge this and test it.